articles/defender-for-cloud/support-matrix-defender-for-servers.md

```diff
@@ -102,6 +102,10 @@ The following table shows feature support for Linux machines in Azure, Azure Arc
 | [Network security assessment](protect-network-resources.md) | ✔ | - | No |
 | [System updates and patches](enable-periodic-system-updates.md) | ✔ | ✔ | Yes (Plan 2)|
 
+> [!NOTE]
+> At present, there is partial feature coverage for VM [Scale Sets with Uniform Orchestration](/azure/virtual-machine-scale-sets/virtual-machine-scale-sets-orchestration-modes#scale-sets-with-uniform-orchestration). Main supported capabilities are agentless detections - such as Network Layer Alerts, DNS alerts, and control plane alerts. Expanded feature coverage for VMSS uniform instances are expected duting 2025. <be>Additionally, there is no support in P1 for the automatic deployment of Microsoft Defender for Endpoint (MDE) on Virtual Machine Scale Sets (VMSS) uniform instances.
```

Comment:
```suggestion
```
This works better as a footnote than in an page note. This way there is a link directly to the text in the document and they don't need to search for where the * leads since it is not really connect.